resolwe_bio/processes/junctions/regtools.yml

```diff
@@ -0,0 +1,119 @@
+  version: 0.0.1
+  type: data:junctions:novel
+  category: analyses
+  flow_collection: sample
+  persistence: CACHED
+  description: >
+    Novel splice junctions in an alignment relative to a reference
+  input:
+    - name: extracted
+      label: Junctions in BED format
+      type: data:junctions:bed
+      description: >
+        Splice junctions previously extracted from an alignment, in BED format. Required if a STAR alignment is not provided.
+      required: false
+    - name: star
+      label: STAR alignment
+      type: data:alignment:bam:STAR
+      description: >
+        STAR alignment, required if a .bed junctions file is not provided. 
+      required: false 
+    - name: gtf
+      label: Reference genome annotation (GTF/GFF3)
+      type: data:annotation
+      description: >
+        A "reference" annotation GTF. 
+  output:
+    - name: bed
+      label: Junctions in BED format
+      type: data:junctions:bed
+      description: Splice junctions in BED format. 
+    - name: sj
+      label: Splice junctions table
+      type: basic:file
+      description: >
+        Table of splice junctions, annotated with read counts, transcript, and gene ID. 
+    - name: novel
+      label: Novel junctions table
+      type: basic:file
+      description: >
+        Filtered Table of annotated novel splice junctions. 
+    - name: stats
+      label: Statistics
+      type: basic:file
+    - name: species
+      label: Species
+      type: basic:string
+    - name: build
+      label: Build
+      type: basic:string
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+      READS_NAME=`basename {{ reads.fastq.0.file }} .fastq.gz`
+      INDEX={{genome.index.dir}}
+
+      FW_READS=()
+      RV_READS=()
+
+
+
+
+- slug: junctions-extract
+  name: Extract splice junctions (regtools)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:regtools
+  data_name: "Extracted Junctions ({{ some input }})"
+  version: 0.0.1
+  type: data:junctions:extracted
+  category: analyses
+  flow_collection: sample
+  persistence: CACHED
+  description: >
+    Extract splice junctions from an alignment using regtools
+  input:
+    - name: bam
+      label: Alignment
+      type: data:alignment:bam
+    - name: gtf
+      label: Reference genome annotation (GTF/GFF3)
+      type: data:annotation
+      description: >
+        A "reference" annotation GTF. 
+  output:
+    - name: bed
+      label: Junctions in BED format
+      type: data:junctions:bed
+      description: Splice junctions in BED format. 
+    - name: species
+      label: Species
+      type: basic:string
+    - name: build
+      label: Build
+      type: basic:string
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+      READS_NAME=`basename {{ reads.fastq.0.file }} .fastq.gz`
+      INDEX={{genome.index.dir}}
+
+      FW_READS=()
+      RV_READS=()
```

Comment:
Missing blank line at the and of file. If you are using Visual Studio Code, you can install `final-newline` extention to check for this automatically.